.github/workflows/deploy.yml

```
@@ -161,3 +161,18 @@ jobs:
       - run: |
           set -a; source ../config/run_envs/${{ inputs.environment }};
           yarn deploy:${{ inputs.environment }}
+
+  apply-algolia-config:
+    name: "apply Algolia settings"
+    if: inputs.environment == 'production'
+    environment: ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: backoffice
+    needs: deploy-helm-release
+    steps:
+      - run: su pcapi -c "flask set_algolia_settings OFFERS set --dry-run=false"
+      - run: su pcapi -c "flask set_algolia_settings COLLECTIVE_OFFERS set --dry-run=false"
+      - run: su pcapi -c "flask set_algolia_settings VENUES set --dry-run=false"
```

Il me semble que la commande a changé et qu'elle n'a plus besoin de sous-commande, désormais.

```suggestion
      - run: su pcapi -c "flask set_algolia_settings OFFERS --dry-run=false"
      - run: su pcapi -c "flask set_algolia_settings COLLECTIVE_OFFERS --dry-run=false"
      - run: su pcapi -c "flask set_algolia_settings VENUES --dry-run=false"
```